cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -427,12 +425,16 @@
               Verify that the CWT is a valid CBOR object.
             </t>
             <t>
-              If the object begins with the CWT CBOR tag, remove it
-	      and verify that one of the COSE CBOR tags follows it.
-	    </t>
-	    <t>
-	      If the object is tagged with one of the COSE CBOR tags, remove it
-	      and verify that it corresponds to the structure of the following COSE object.
+              If the object begins with the CWT CBOR tag, remove it and
+              continue verification.
+            </t>
+            <t>
+              If the object is tagged with one of the COSE CBOR tags, remove it
+              and use it to determine the type of the CWT,
+              COSE_Sign/COSE_Sign1, COSE_Mac/COSE_Mac0, or
+              COSE_Encrypt/COSE_Encrypt0.
+              If object does not have a have a COSE CBOR tag the type has to be
+              determine based on the application context.
```

Comment:
Change this sentence to "If the object does not have a COSE CBOR tag, the COSE message type is determined from the application context."